### ISSUE TITLE
Title with RandomGender generates firstname instead of Title

### ISSUE THREAD

**Issue by marcofranssen** [Pallinder/go-randomdata#60]
See below the bug in default switch.

```
// Title returns a random title, gender decides the gender of the name
func Title(gender int) string {
	var title = ""
	switch gender {
	case Male:
		title = randomFrom(jsonData.MaleTitles)
		break
	case Female:
		title = randomFrom(jsonData.FemaleTitles)
		break
	default:
		title = FirstName(privateRand.Intn(2))
		break
	}
	return title
}
```

**Comment by Pallinder** [comment]
Thanks for this! Think you could open a PR so we can fit into release 1.2.0?

**Comment by daisy1754** [comment]
Hi @Pallinder  I've opened #64 to fix this. Do you mind taking a look (also it'd be great if you can look at #63 too :) )

**Comment by Pallinder** [comment]
Released in v1.2.0
